modules/ROOT/pages/constraints/syntax.adoc

```diff
@@ -69,8 +65,6 @@ REQUIRE (r.propertyName_1, ..., r.propertyName_n) IS [REL[ATIONSHIP]] UNIQUE
 [OPTIONS "{" option: value[, ...] "}"]
 ----
 
-An index provider can be specified using the `OPTIONS` clause.
```

Comment:
Like I said on some other comment I wasn't sure I could remove them since the language still supports to have an options map. But I'd be happy to see it go if we decide it's okay to remove it
I give cypher and docs team free reign to change whatever on this PR :)

Reply:
I like option 1 👍